core/src/test/java/org/sonarsource/sonarlint/core/telemetry/TelemetryManagerTest.java

```diff
@@ -366,6 +377,17 @@ public void uploadLazily_should_clear_accumulated_data() {
     assertThat(reloaded.taintVulnerabilitiesInvestigatedRemotelyCount()).isZero();
   }
 
+  @Test
```

Comment:
you should update test `uploadLazily_should_clear_accumulated_data` to ensure `raised_issues` are purged after ping is sent

Reply:
Done.